lib/YaST/workarounds.pm

```diff
@@ -8,10 +8,11 @@ use strict;
 use warnings;
 use Exporter 'import';
 use testapi;
+use utils;
 use version_utils;
 
 
-our @EXPORT = qw(apply_workaround_bsc1204176);
+our @EXPORT = qw(apply_workaround_bsc1204176 apply_workaround_bsc1207157);
```

Comment:
nitpick: just `qw(` at the end of this line, so the values of the list are aligned below.